### PR TITLE
Update lecture material paths and extra tag handling in the update_workbook script.

### PR DESCRIPTION
**Pull request by YooSunYoung**

### CODE REVIEW

File: update_workbook.py [hunk 1]
```diff
@@ -157,13 +163,17 @@ def _create_workbook_contents(self) -> dict:
 
         How to create a workbook from a textbook
         ----------------------------------------
-        For all cells with ``solution`` tag,
-        remove ``source`` of the cell and ``hide-cell``, ``solution`` tags and
-        add ``SOURCE_REPLACEMENTM_MESSAE`` to the ``source``.
+        For all cells without ``dmsc-school-keep`` tag (``CELL_PROTECTING_TAG``).
+        1. Remove all cells containing ``remove-cell``.
+        2. Replace source code with ``SOURCE_REPLACEMENTM_MESSAE``
```
Comment (nvaytet) [inline update_workbook.py:168]:
```suggestion
        2. Replace source code with ``SOURCE_REPLACEMENT_MESSAGE``
```

File: update_workbook.py [hunk 2]
```diff
@@ -27,21 +28,21 @@ class GitStatus(Enum):
...
 
 lecture_to_directories = MappingProxyType({
-    Lecture.MCSTAS: Path('mcstas'),
-    Lecture.MODEL: Path('model'),
-    Lecture.PROPOSALS: Path('proposals'),
-    Lecture.PYTHONBASICS: Path('python'),
-    Lecture.SCICAT: Path('scicat'),
-    Lecture.SCIPP: Path('scipp'),
+    Lecture.PYTHONBASICS: Path('1-python'),
+    Lecture.PROPOSALS: Path('2-proposals'),
```
Comment (nvaytet) [inline update_workbook.py:41]: this folder is now gone. Will the script break if the folder is not found or will it happily carry on?

Comment (YooSunYoung) [reply update_workbook.py:41]: Then they will be ignored.
It's for filtering the directories that matter.

**Comment by YooSunYoung** [comment]
@nvaytet 
I think this script should be used only for `notebooks` and `python` modules/scripts, since it's not only copying but it is about updating the contents of the jupyter notebooks. But I included images at least.

Then other files can be copied with your script maybe...?

**Pull request merged**